src/Componenets/Country.js

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable import/extensions */
+import React, { useEffect, useState } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import './Country.css';
+import { FaMicrophone } from 'react-icons/fa';
+import { AiFillSetting } from 'react-icons/ai';
+import { fetchCountry } from '../Redux/slices/CountrySlice';
```

Comment:
- [ ] The same happens here:

```suggestion

import React, { useEffect, useState } from 'react';
import { useParams, useNavigate } from 'react-router-dom';
import { useSelector, useDispatch } from 'react-redux';
import './Country.css';
import { FaMicrophone } from 'react-icons/fa';
import { AiFillSetting } from 'react-icons/ai';
import { fetchCountry } from '../Redux/Slices/CountrySlice';
```